fmriprep/cli/parser.py

```diff
@@ -541,12 +534,18 @@ def _bids_filter(value):
 
 def parse_args(args=None, namespace=None):
     """Parse args and run further checks on the command line."""
+    import os
     import logging
     from niworkflows.utils.spaces import Reference, SpatialReferences
     from niworkflows.utils.misc import check_valid_fs_license
 
     parser = _build_parser()
     opts = parser.parse_args(args, namespace)
+
+    if os.getenv('FMRIPREP_NO_REUSE_CONFIG') is not None:
+        # reuse config if available
+        config.load_previous_config(opts.output_dir, opts.work_dir, opts.participant_label)
+
     config.execution.log_level = int(max(25 - 5 * opts.verbose_count, logging.DEBUG))
     config.from_dict(vars(opts))
```

Comment:
This will rewrite all the options from the command line. Now that we removed opts.run_uuid, this should be safe - can you have a deep look into what should/can/may be overwritten here?